cointop/table_header.go

```diff
@@ -95,6 +95,11 @@ var HeaderColumns = map[string]*HeaderColumn{
 		Label:      "[3]0D%",
 		PlainLabel: "30D%",
 	},
+	"1y_change": &HeaderColumn{
+		Slug:       "1y_change",
+		Label:      "3[6]5D%",
```

Comment:
I don't like this keyboard shortcut because it's more common to use 1Y instead of 365D. 

What do you think about using `1[Y]%` (y or Y)  instead?

Reply:
Totally happy to change it. Go for it - I'm not on the right computer to make changes to simon-anz right now.

Reply:
OK, change pushed.